docs/core/tutorials/publishing-with-visual-studio.md

```diff
@@ -1,49 +1,67 @@
 ---
-title: Publish your .NET Core Hello World application with Visual Studio 2017
+title: Publish your .NET Core Hello World application with Visual Studio
 description: Publishing creates the set of files that are needed to run your .NET Core application.
 author: BillWagner
 ms.author: wiwagn
-ms.date: 10/05/2017
+ms.date: 12/10/2019
 ms.custom: "vs-dotnet, seodec18"
 ---
-# Publish your .NET Core Hello World application with Visual Studio 2017
+# Publish your .NET Core Hello World application with Visual Studio 2019
 
-In [Build a Hello World application with .NET Core in Visual Studio](with-visual-studio.md), you built a Hello World console application. In [Debug your Hello World application with Visual Studio](debugging-with-visual-studio.md), you tested it using the Visual Studio debugger. Now that you're sure that it works as expected, you can publish it so that other users can run it. Publishing creates the set of files that are needed to run your application, and you can deploy the files by copying them to a target machine.
+In [Build a Hello World application with .NET Core in Visual Studio](with-visual-studio.md), you built a Hello World console application. In [Debug your Hello World application with Visual Studio](debugging-with-visual-studio.md), you tested it using the Visual Studio debugger. Now that you're sure that it works as expected, you can publish it so that other users can run it. Publishing creates the set of files that are needed to run your application. To then deploy the files, copy them to the target machine.
```

Comment:
Suggest dropping "then" in the final sentence.
```suggestion
In [Build a Hello World application with .NET Core in Visual Studio](with-visual-studio.md), you built a Hello World console application. In [Debug your Hello World application with Visual Studio](debugging-with-visual-studio.md), you tested it using the Visual Studio debugger. Now that you're sure that it works as expected, you can publish it so that other users can run it. Publishing creates the set of files that are needed to run your application. To deploy the files, copy them to the target machine.
```